### PR TITLE
Make value nameOverride overide name completely

### PR DESCRIPTION
**Pull request by pulledtim**
The value "nameOverride" should set the complete name of the deployment/service, not just a part of it.
#2772

### CODE REVIEW

File: stable/gcloud-sqlproxy/templates/_helpers.tpl [hunk 1]
```diff
@@ -11,6 +11,6 @@ Create a default fully qualified app name.
 We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
 */}}
 {{- define "gcloud-sqlproxy.fullname" -}}
-{{- $name := default .Chart.Name .Values.nameOverride -}}
-{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- $name := .Values.nameOverride | default (printf "%s-%s" .Release.Name .Release.Name) -}}
```
Comment (unguiculus) [inline stable/gcloud-sqlproxy/templates/_helpers.tpl]: `nameOverride` is meant for the `name` template. Either add a `fullnameOverride` or do something like this: https://github.com/kubernetes/charts/blob/master/incubator/oauth-proxy/templates/_helpers.tpl#L14-L18

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by unguiculus** [comment]
Also marking stale. Please fix your CLA.

**Comment by pulledtim** [comment]
@unguiculus  Thank you for your input. I added a new parameter fullnameOverride as you proposed. CLA should have been signed now by employer, will find out why its not shown yet.

**Review by unguiculus — APPROVED** [review]

**Pull request merged**